java/ql/automodel/src/README.md

```diff
@@ -0,0 +1,183 @@
+# Automodel Java Extraction Queries
+
+This pack contains the automodel extraction queries for Java.
+
+## Extraction Queries in `java/ql/automodel/src`
+
+This pack contains extraction queries for application mode and framework mode.
+
+| Kind | Mode | Query File |
+|------|------|------------|
+| Candidates | Application Mode | [AutomodelApplicationModeExtractCandidates.ql](https://github.com/github/codeql/blob/main/java/ql/automodel/src/AutomodelApplicationModeExtractCandidates.ql) |
+| + Examples | Application Mode | [AutomodelApplicationModeExtractPositiveExamples.ql](https://github.com/github/codeql/blob/main/java/ql/automodel/src/AutomodelApplicationModeExtractPositiveExamples.ql) |
```

Comment:
Minor: I find `+ Examples` slightly hard to grok. I know you don't want the columns too wide. Can you do `Pos Examples` and `Neg Examples`? Or add a newline between `Postive` and `Examples`?

Reply:
I've been using this term, as we often use this in written conversations. In this case, we have the space — I've replaced the +/- by positive/negative throughout the document!